Documentation/Browser/AddNewIDLFile.md

```diff
@@ -0,0 +1,31 @@
+# Adding a new IDL file
+
+Serenity's build system does a lot of work of turning the IDL from a Web spec into code, but there are a few things you'll need to do yourself.
+
+For the sake of example, let's say you're wanting to add the `HTMLDetailsElement`.
+
+1. Create `LibWeb/HTML/HTMLDetailsElement.idl` with the contents of the IDL section of the spec. In this case, that would be:
+```
+[Exposed=Window]
+interface HTMLDetailsElement : HTMLElement {
+  [HTMLConstructor] constructor();
+
+  [CEReactions] attribute boolean open;
+};
+```
+
+2. If the IDL starts with `[Exposed=Window]`, remove that line from the .idl file, and add the following to `LibWeb/Bindings/WindowObjectHelper.h`:
```

Comment:
I like to link files from markdown files, many editors as well as GitHub support it. If you look at this doc in the browser you can just open all of them in a new tab easily to have a look.

```suggestion
2. If the IDL starts with `[Exposed=Window]`, remove that line from the .idl file, and add the following to [`LibWeb/Bindings/WindowObjectHelper.h`](../../Userland/Libraries/LibWeb/Bindings/WindowObjectHelper.h):
```